data/campaigns/The_South_Guard/scenarios/01_Born_to_the_Banner.cfg

```diff
@@ -652,6 +655,27 @@
             animate=yes
         [/kill]
 
+        [if]
+            [not]
+                [have_unit]
+                    id="Sir Gerrick"
+                [/have_unit]
+            [/not]
+            [then]
+                {SIR_GERRICK}
```

Comment:
This is because it's possible to get here without him having been found. As I said, a WIP. I think this will get yanked from here because I did this before I added the scenario_end clean-up to ensure he's there when moving forward in the campaign.